azext_edge/edge/common.py

```diff
@@ -214,6 +214,36 @@ class AEPAuthModes(Enum):
     userpass = "UsernamePassword"
 
 
+class AEPTypes(Enum):
+    """Asset Endpoint Profile (connector) Types"""
+    # TODO: ensure this is the final enum
+    opcua = "OpcUa"
```

Comment:
why enum instead of constant string if there is only one value? will there be more coming?

Reply:
yes - more to come at some point.